doc/sphinx/data_notes.rst

```diff
@@ -117,6 +117,13 @@ coreelectrons
 
 The attribute ``coreelectrons`` contains the number of core electrons in each atom's pseudopotentials. It is an array of rank 1, with as many integer elements as there are atoms.
 
+dispersionenergies
+------------------
+
+This is a rank 1 array that contains the isolated dispersion energy for each geometry.  This will be generally true for empirical models, such as those from Grimme that only depend on relative atomic positions.  For self-consistently determined corrections that are incorporated during SCF iterations, this array will be empty.
```

Comment:
Is the phrasing of "this is generally true" indicating that the array will be populated for methods like Grimme? If this is phrasing I'm not familiar with, just let me know. Otherwise, stating it is populated instead of true would be more clear.

Reply:
The former, so I fixed it.